### DISCUSSION TITLE
Off-Topic • Delete My Account

### DISCUSSION THREAD

**Discussion by bettywolfy** [nesbox/TIC-80#2178] [General]
Is this the best place to talk about this?

I would like to recreate my website account

Could you delete it?

**Comment by Anrock** [comment]
@nesbox

**Comment by nesbox** [comment]
There is no such functionality, I can only delete it manually in the DB.
But you have to prove somehow you are the owner of the account.

**Comment by Anrock** [comment]
@nesbox this isn't the first account deletion request in my memory. Could it be useful to document some kind of procedure for deletion? Something along the lines of
1) Publish a simple cartridge with account deletion approval (just clone demo one and replace text)
2) Create a ticket / contact via email

**Comment by nesbox** [comment]
I think "Publish a simple cartridge with account deletion approval" is a good idea.
@EliasWolf-Br could you pls publish the cart?

**Reply by bettywolfy** [reply]
Dude, my account has a "pls delete" on icon

Is this not enough? 😅

**Reply by bettywolfy** [reply]
Ok, ok...

**Reply by bettywolfy** [reply]
![Screenshot_20230508-122811](https://user-images.githubusercontent.com/92587199/236878324-ee4c7c38-870b-40a9-a81e-1f6a179563a8.png)

A game on my new account: https://tic80.com/play?cart=3334

**Reply by bettywolfy** [reply]
@nesbox ✓?

**Comment by nesbox** [comment]
done!
